test/functional/conftest.py

```diff
@@ -268,6 +268,14 @@ def get_docker_executable():
     return distutils.spawn.find_executable('docker')
 
 
+def get_linode_executable():
+    try:
+        pytest.importorskip('linode')
+        return True
+    except Exception:
```

Comment:
What exception is actually happening here? AFAIK `pytest.importorskip` with just return `None` if there's nothing to import. Which means that this check'd always return `True`.

`pytest.importorskip` is specifically designed to trigger skipping the current test anyway so I don't know why you would wrap it like this.

Reply:
Also, a bunch of unrelated changes makes it difficult to guess what is the purpose of the PR...

Reply:
Oh.. I see now.

Reply:
I'm not sure how this check works but it's probably not related to the scope of the current PR. So let's ignore it for now.

Reply:
I just wanted to align on the other platforms where there is a split between def_xxx_executable and a supports_xxx.
The try and exception is just a copy paste during the split, nothing new here.

Reply:
Yes, I noticed that already and suggested to address it separately in other comments.